lib/kredis/types/counter.rb

```diff
@@ -3,6 +3,10 @@ class Kredis::Types::Counter < Kredis::Types::Proxying
 
   attr_accessor :expires_in
 
+  def callback_operations
+    %i[increment decrement reset].freeze
+  end
```

Comment:
This feels like a configuration-level method that shouldn't be exposed directly through the API. What about using a constant instead?

Reply:
Then you also don't have a to call freeze. And for style, space after opening and before closing 👍

Reply:
Makes sense!

Reply:
I'm too used to `standard` 😅